apps/nextjs-website/react-components/types/Editorial-Switch/Editorial-Switch.types.ts

```diff
@@ -2,7 +2,7 @@ import { EditorialProps } from '../Editorial/Editorial.types';
 
 export interface EditorialSwitchBaseProps {
   title: string;
-  subtitle?: string;
+  body?: string | JSX.Element | null;
```

Comment:
```suggestion
  subtitle?: JSX.Element;
```
We want `subtitle` to always be rendered frrom markdown. Accept only JSX.Element